app/views/spree/admin/payments/source_forms/_braintree.html.erb

```diff
@@ -0,0 +1,64 @@
+<fieldset data-id='credit-card' class="no-border-bottom">
```

Comment:
whats modified here from the original ```_gateway.html.erb```? If we can work to reduce the duplication and better override the specifics for braintree I think we'll have more success in the long term.